### DISCUSSION TITLE
对于鼠标所在的那句话，同时高亮原文和译文

### DISCUSSION THREAD

**Discussion by yufengwhy** [immersive-translate/immersive-translate#2219] [Ideas]
### 新功能 | New Feature

1.
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.翻译不准确或者奇怪的时候，需要回看原文，但是现在段落里面找到那句话比较费时间
2.
3.

**Comment by theowenyoung** [comment]
您好，目前沉浸式翻译使用第三方翻译服务，无法做到精确到句子的定位。

**Comment by yufengwhy** [comment]
@theowenyoung 感谢回复，相信句子级别高亮这个实用功能肯定能给我们带来更多的用户或付费。需要的话也可以给我一个入口，我研究下提个merge。  
双语句子对齐可以加一些者后处理（sentence align）或者前处理（sentence translate）来实现？   
句子高亮可以加一些css控制？  

Reference:  

> 现成的句子对齐工具:
Hunalign: 开源的句子对齐工具
awesome-align: 基于BERT的对齐工具
LaBSE: Google的多语言句向量模型,可用于相似度计算

**Reply by theowenyoung** [reply]
不过我想补充一下，即使我们最终技术上能实现这个，沉浸式翻译在功能上也没有计划做这个特性～
